hr_operating_unit/models/hr_employee.py

```diff
@@ -0,0 +1,20 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    operating_unit_ids = fields.Many2many(
+        'operating.unit', 'operating_unit_employees_rel', 'employee_id',
+        'poid', 'Operating Units',
```

Comment:
@meigallodixital Name should be poid --> `operating_unit_id`

Reply:
I have called it **poid** to maintain the nomeclature of the operating unit base module [1]. I have no problem to change it.

[1] https://github.com/OCA/operating-unit/blob/91cfccae6101e9d4e90825a9d6484c9c25ada786/operating_unit/models/res_users.py#L28

Reply:
Yes, sorry about that, will be fixed in v13. Now it is not convenient to change that. There is no benefit and there is a migration script needed. Not worth it now.

Reply:
Do we confirm then that for 12.0 we keep 'poid' then?

We have also migrated hr_holidays and  ht_attendance. If we close this PR we send the other two. Same github repository on its branches.

As this PR is too old (commits are from May) could it be good to rebase and squash cmmits and send it again?

Reply:
Yes, keep poid in this version. Will be fixed in v13 with the proper migration script.

Regarding the other question. PRs will be accepted if they solve a generic use case that is not covered by the existing modules and the PR makes sense. For example, for holidays, the operating unit must be related to the employee. We need to make sure we don't create inconsistencies between the different models. But, of course, any contribution is welcome. :)